src/Dict.cc

```diff
@@ -12,6 +12,196 @@ namespace zeek
 
 TEST_SUITE_BEGIN("Dict");
 
+//to show outputs of unit tests, define PRINTF as fprintf by uncommenting fprintf.
+//and run ./zeek --test -tc="dict verify clone performance"
+#define PRINTF(fmt, args...) //fprintf(stderr, fmt, args)
+
+struct ReportDuration
+	{
+	std::chrono::_V2::system_clock::time_point start;
+	std::chrono::_V2::system_clock::time_point stop;
```

Comment:
These two lines are causing build failures on a number of the CI hosts. On a local build, changing it to `std::chrono::high_resolution_clock::time_point` appears to fix it.

Reply:
changed accordingly